src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -734,6 +734,7 @@ class ApplicationManagerImpl
   bool CanStartProtectedService(
       const int32_t& session_key,
       const protocol_handler::ServiceType& type) const OVERRIDE;
+  bool IsNaviApp(const int32_t& session_key) const OVERRIDE;
```

Comment:
@tkireva What do you think about changing name to HasNaviApp (or something similar) that clearly points that this particular session owns navi app.